go/staking/api/api.go

```diff
@@ -1183,6 +1183,13 @@ type Genesis struct {
 
 // ConsensusParameters are the staking consensus parameters.
 type ConsensusParameters struct { // nolint: maligned
+	// TokenSymbol is the token's ticker symbol.
+	// Only upper case A-Z characters are allowed.
+	TokenSymbol string `json:"token_symbol,omitempty"`
```

Comment:
When/where will these two fields be set?

Reply:
Currently only via an upgrade handler. I would keep it this way for now.